libraries/stdlib/src/kotlin/io/FileReadWrite.kt

```diff
@@ -196,6 +192,5 @@ public fun File.readLines(charset: Charset = Charsets.UTF_8): List<String> {
  * @param charset character set to use. By default uses UTF-8 charset.
  * @return the value returned by [block].
  */
-@RequireKotlin("1.2", versionKind = RequireKotlinVersionKind.COMPILER_VERSION, message = "Requires newer compiler version to be inlined correctly.")
```

Comment:
This annotation originates from `use` function and is contagious for all inline functions in the standard library, so it would be incorrect to remove it here.

Reply:
I got it. Removed this because `use` was changed to `useLines` and the latter has the proper annotation on itself.